src/utils/daos_hdlr.c

```diff
@@ -2734,6 +2734,13 @@ dm_connect(bool is_posix_copy,
 		}
 	}
 
+    /* set cont_layout to POSIX type if the source is not in DAOS, if the 
+     * destination is DAOS, and no destiantion container exists yet,
+     * then it knows to create a POSIX container
+     */
+    if (src_file_dfs->type == POSIX) 
```

Comment:
(style)  trailing whitespace
(style)  suspect code indent for conditional statements (4, 12)